scripts/scripts.js

```diff
@@ -129,6 +129,7 @@ const CONFIG = {
   htmlExclude: [
     /business\.adobe\.com\/(\w\w(_\w\w)?\/)?blog(\/.*)?/,
   ],
+  useDotHtml: true,
```

Comment:
I'm assuming this config will force the use of the `appendHtmlToCanonicalUrl` [1] and `appendHtmlToLink` [2] methods or possibly add an additional check to the `DOT_HTML_PATH` constant [3] in Milo?

[1] https://github.com/adobecom/milo/blob/main/libs/utils/utils.js#L273
[2] https://github.com/adobecom/milo/blob/main/libs/utils/utils.js#L284
[3] https://github.com/adobecom/milo/blob/main/libs/utils/utils.js#L119

Reply:
Yes see https://github.com/adobecom/milo/pull/1148